config/config.exs

```diff
@@ -24,6 +24,10 @@ config :keila, Keila.Id,
   salt: "bF4QzDjqV",
   min_len: 8
 
+config :keila, Keila.Mailings,
+  # Minimum offset in seconds between current time and allowed scheduling time
+  campaign_schedule_offset: 300
```

Comment:
I wonder if there would be a more descriptive name we could use but I honestly can't reallt come up with anything better on the spott.

Reply:
I went through three iterations or so to come up with the name, but it’s really hard to describe the concept - so I’m open for better suggestions :smile: 
Maybe `min_campaign_schedule_offset` would be a bit clearer?

Reply:
Yeah `min_campaign_schedule_offset` sounds like good enough 👍🏻

Reply:
:partying_face: